libclamav/ole2_extract.c

```diff
@@ -1703,7 +1679,7 @@ static cl_error_t handler_otf(ole2_header_t *hdr, property_t *prop, const char *
                 ret = CL_EUNLINK;
             }
         }
-        free(tempfile);
+        FREE(tempfile);
```

Comment:
I think this will generate a warning because we've already checked if its not NULL and we've dereferenced it.  I would keep this as a regular `free()`.